lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+class Menu
+  attr_reader :food
+
+  def initialize(food)
+    @food = food
+  end
+
+  def print_menu
+    x = food.map do |item, price|
+      "%s £%.2f" % [item.to_s.capitalize, price]
```

Comment:
What does this do? "%s £%.2f" - looks like it shows your price as £x.xx?

Reply:
Correct! It changes the string format to one which is actual pricing, the %.2f is important as otherwise prices would be returned as a float with 1 decimal place (i.e. £5.5) which isn't technically correct